### ISSUE TITLE
Send messages with Enter key

### ISSUE THREAD

**Issue by Loirooriol** [mozillahispano/openwapp#161]
I use OpenWapp on a Firefox OS Simulator on a PC. So I write messages using the physical keyboard.

The problem is that OpenWapp requires me to click the send button with the mouse in order to send the message. And that's annoying.

Instead, I want to be a able to send by pressing the Enter key. So I added the following code to `compose-message.js`

```javascript
events: {
  // ...
  'keydown #message-text-input' : '_sendOnEnter',
},
_sendOnEnter: function (e) {
  if(e.key === 'Enter') {
    this.$el.find('#conversation-send-button').get(0).click();
    e.preventDefault();
  }
}
```

But of course it would be better if OpenWapp had a checkbox in the config section to choose the desired behavior.

**Comment by Timmaks** [comment]
can you make a PR of this and add  a checkbox in the profile?

**Comment by Loirooriol** [comment]
I thought about that. But how do I store the preference? `localStorage`? Does OpenWapp have some API to manage preferences?

**Comment by delapuente** [comment]
Modify this model: https://github.com/mozillahispano/openwapp/blob/master/app/scripts/models/auth.js

**Comment by Loirooriol** [comment]
@delapuente Thanks for the link, but that `authStorage` seems asynchronous, and I needed to get the preference synchronously in order to decide whether the event should be cancelled.

I saw the `awakePeriod` preference is synchronously stored through `global.backgroundService`, but that didn't seem a good place to store my send-by-enter pref.

Since `backgroundService` stores `awakePeriod` using `localStorage` under the hood, at the end I used `localStorage` directly.
